### ISSUE TITLE
About pretraining process

### ISSUE THREAD

**Issue by sunanhe** [salesforce/ALBEF#38]
First of all, thanks for the great work.
I use 8 V100 to train the model with 4M images. it takes 5~6 hours for an epoch which is too slow. is it correct?
Could you please share the pretraining log, I want to check the loss of the pretraining process.
Thank you in advance.

**Comment by LiJunnan1992** [comment]
You can try to use zero-redundancy optimizer and gradient checkpointing (https://fairscale.readthedocs.io/en/latest/tutorials/oss.html) to increase the batch size, which might increase your training speed.

**Comment by sunanhe** [comment]
Thanks for your reply！
